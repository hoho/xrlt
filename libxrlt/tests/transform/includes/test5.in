id:0, type:0, last:0, error:0, data:
id:0, type:0, last:0, error:0, data:
id:1, type:400, last:0, error:0, data:200
id:1, type:600, last:0, error:0, data:hello=wo%20r
id:1, type:600, last:0, error:0, data:ld&this=is&this
id:1, type:600, last:1, error:0, data:=IS;very=co%20ol&blah;=he;=ha
id:0, type:0, last:0, error:0, data:
