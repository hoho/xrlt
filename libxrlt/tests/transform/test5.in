id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:1, type:2, last:0, error: 0, data:hello=wo%20r
id:1, type:2, last:0, error: 0, data:ld&this=is&this
id:1, type:2, last:1, error: 0, data:=IS;very=co%20ol&blah;=he;=ha
id:0, type:0, last:0, error: 0, data:
