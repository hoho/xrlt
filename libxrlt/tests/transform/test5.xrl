<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">
    <xrl:response>
        <xrl:include>
            <xrl:href>/test/href/1</xrl:href>
            <xrl:type>querystring</xrl:type>
       </xrl:include>
    </xrl:response>
</xrl:requestsheet>
