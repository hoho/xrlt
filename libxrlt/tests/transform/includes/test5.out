XRLT_STATUS_SUBREQUEST
sr id: 1
sr method: GET
sr type: QUERYSTRING
sr url: /test/href/1
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: wo rld
chunk: is
chunk: IS
chunk: co ol
chunk: heha
XRLT_STATUS_DONE
